src/main/java/com/github/tomakehurst/wiremock/standalone/CommandLineOptions.java

```diff
@@ -387,7 +389,23 @@ public MappingsSaver mappingsSaver() {
 
     @Override
     public Notifier notifier() {
-        return new ConsoleNotifier(verboseLoggingEnabled());
+        String notifier = (String) optionSet.valueOf(NOTIFIER);
+        if ("ConsoleNotifier".equals(notifier)) {
```

Comment:
I think that you should use ConsoleNotifier.class.getSimpleName() to avoid changes in future.